python/raydp/torch/torch_metrics.py

```diff
@@ -0,0 +1,44 @@
+import torchmetrics
+import sys
+module = sys.modules[__name__]
+
+class Torch_Metric():
+    def __init__(self, metrics_name, metrics_config):
+        self._metrics_name = metrics_name
+        self._preprocess_fun = {}
```

Comment:
What about rename `fun` to `func` in all places?

Reply:
thanks. done.